### PR TITLE
[Metricbeat] add windows.service.start_name and windows.service.path_name

### PR DESCRIPTION
**Pull request by jcallaha**
This PR implements gathering the executable path name of a service and security principal used when the service is started. Based on the idea in issue #8364.  I chose the field names based on WMI to be consistent.

Can be useful in discovering rogue services or services configured with different security principals on different systems.

### CODE REVIEW

**Comment by willemdh** [comment]
@jcallaha Thanks for this PR :) 

Just something to discuss. I can understand you want to match the field names to the names in WMI, but it might be a more future proof idea to try follow ecs guidelines? https://github.com/elastic/ecs

What do you think @ruflin ?

**Comment by ruflin** [comment]
You will need to run `make update` to get this green.

For ECS: We plan to go through all modules and convert them to ECS. For the fields added here I would keep them as is for now.

Can you please also add a CHANGELOG entry?

**Comment by jcallaha** [comment]
I hope I did the CHANGELOG correctly.  Let me know if there is anything else.  Thanks!

**Comment by sayden** [comment]
@jcallaha you also need to add the PR number in the CHANGELOG (there are some examples in the same file) plus fix the conflict and rebase the branch. To fix the conflict just leave as much information as possible.

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -43,8 +45,10 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 - Replace index patterns in TSVB visualizations. {pull}7929[7929]
 - Fixed Support `add_docker_metadata` in Windows by identifying systems' path separator. {issue}7797[7797]
 - Add backoff support to x-pack monitoring outputs. {issue}7966[7966]
-- Fix a race condition with the `add_host_metadata` and the event serialization. {pull}8223[8223]
+- Fix a race condition with the `add_host_metadata` and the event serialization. {pull}8223[8223] {pull}8653[8653]
 - Enforce that data used by k8s or docker doesn't use any reference. {pull}8240[8240]
+- Switch to different UUID lib due to to non-random generated UUIDs. {pull}8485[8485]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: I think something went wrong during rebase. Lots of entries here which should not be here.

**Comment by ruflin** [comment]
@jcallaha I think something went wrong a bit during rebasing. It seems you have a lot of changes in this PR which are already in master. Can you try to rebase again on the most recent version of master?

**Comment by jcallaha** [comment]
@ruflin Sorry about that, I'm working in Visual Studio Code and must have done something unexpected through the UI.  I redone the rebase with a forced push using just the git command line and things seem to look better now.

**Comment by ruflin** [comment]
@andrewkroh Could you perhaps have a look at this PR?

**Comment by ruflin** [comment]
jenkins, test this

**Comment by andrewkroh** [comment]
jenkins, test this

**Comment by narph** [comment]
jenkins, test this

**Comment by narph** [comment]
@jcallaha , do you the time to resolve the conflicts on this PR? We would like to pull this in. If you don't have the time I'll cherry-pick your commit and happy to resolve the conflicts myself.

**Comment by jcallaha** [comment]
@narph sorry looks like I can't easily resolve the conflicts without setting up a build environment again to generate metricbeat/module/windows/fields.go - please go ahead and cherry-pick and resolve.  Thanks!

**Comment by narph** [comment]
@jcallaha , np, I opened https://github.com/elastic/beats/pull/11877 to get the ball rolling again, thank you

**Comment by narph** [comment]
closing in favor of merged PR https://github.com/elastic/beats/pull/11877

**Pull request closed without merging**